server_py/flatgov/bills/templatetags/bill_filters.py

```diff
@@ -30,7 +30,7 @@ def billnumber_display(billnumber: str):
         number = billMatchGroups.get('number', '')
         stage_fmt = stagesFormat.get(stage.upper(), stage)
         if congress != constants.CURRENT_CONGRESS:
-           billnumber_fmt = f' {stage_fmt} {number} ({congress})'
+           billnumber_fmt = f' {stage_fmt} {number}'
```

Comment:
This code supposed to put the congress in parentheses if it is not the current congress. With the change, we will never get the ({congress}) parenthetical.